.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+jobs:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
```

Comment:
Not sure what fast_finish does, but its suggested by Travis. Also we don't want nightly failures to prevent PRs to be merged so we ignore those failures.